nmmo/task/utils.py

```diff
@@ -0,0 +1,48 @@
+from typing import List
+from nmmo.task.group import Group
+
+class TeamHelper:
```

Comment:
TeamHelper in baselines is gone, but we still need something like this in nmmo env for left/right etc